cm_channels/cm_formationChannels.py

```diff
@@ -63,12 +62,11 @@ def register(self, objs):
 
     def newuser(self, userid):
         """Called when a new agent is using this channel"""
-        self.userid = userid
+        self.userid = userid  # should be defined in __init__!
```

Comment:
Go ahead if you want to. It's guaranteed to not be used before this method is called but could be initialised to "" or None.